### ISSUE TITLE
Always ignore the build tags

### ISSUE THREAD

**Issue by rakyll** [sqs/goreturns#33]
goreturns do not ignore the build tags, and causing editor plugins to skip formatting the Go files under a build tag (https://github.com/Microsoft/vscode-go/issues/877). 

goreturns is an editing tool and should always ignore the build tags.

**Comment by sqs** [comment]
Agreed. I think this is a quick fix. I will take a look this weekend. If anyone else wants to submit a PR in the meantime, I would happily merge it.
